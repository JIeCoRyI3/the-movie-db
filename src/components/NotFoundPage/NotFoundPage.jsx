import React from 'react';
import Header from '../Header/';
import NotFound from '../NotFound';
import SearchBar from '../Header/Mocks/MockSearchBar';

function NotFoundPage() {
    return (
        <>
            <Header>
                <SearchBar />
            </Header>
            <NotFound />
        </>
    );
}

export { NotFoundPage };
