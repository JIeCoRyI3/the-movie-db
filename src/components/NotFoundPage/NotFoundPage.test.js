import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from "../NotFoundPage";


describe('< NotFoundPage /> render', () => {
	it('should render component',  () => {
		const component = shallow( <NotFoundPage />);
		expect(component).toMatchSnapshot();
	})
})
