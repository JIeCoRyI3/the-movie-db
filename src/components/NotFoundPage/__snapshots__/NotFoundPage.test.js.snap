// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< NotFoundPage /> render should render component 1`] = `
<Fragment>
  <Connect(Header)>
    <Connect(SearchBar) />
  </Connect(Header)>
  <NotFound />
</Fragment>
`;
